src/main/java/com/salesforce/bazel/javabuilder/mybuilder/MyBuilderProcessor.java

```diff
@@ -1,5 +1,58 @@
 package com.salesforce.bazel.javabuilder.mybuilder;
 
-public class MyBuilderProcessor {
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.io.*;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.Callable;
+
+@Command(name="builder-template", description = "builder template")
+public class MyBuilderProcessor implements Callable<Void> {
+
+	@Option(names = { "-o", "--outputFile" }, description = "The Output File")
+    private String outputFile;
+
+    @Option(names = { "-i", "--inputFile" }, description = "The Input File")
+    private String inputFile;
+
+    public static void main(String[] args) throws IOException {
+        CommandLine.call(new MyBuilderProcessor(), args);
+    }
+
+    @Override
+    public Void call() throws Exception {
+    	MyBuilderProcessor mybuilder = new MyBuilderProcessor();
+    	//inputs to be processed here
+    	mybuilder.copyToFile(inputFile, outputFile);
+    	return null;
+    }
+
+    private Set<String> getInformationFromFile(String filePath) throws IOException {
+        File file = new File(filePath);
+        BufferedReader br = new BufferedReader(new FileReader(file));
```

Comment:
Use try-with-resources. Have a look at static utility methods in JDK `Files` class.